### PR TITLE
HACKING: clear up notes around development pipelines

### PR DESCRIPTION
**Pull request by dustymabe**
Make it clear that developer pipelines are to be run on existing
clusters/deployments. Also add more context and notes about defaults
and existing behavior.

### CODE REVIEW

**Comment by dustymabe** [comment]
still testing this out to make sure there isn't any small tweaks needed before merging

**Comment by dustymabe** [comment]
> still testing this out to make sure there aren't any small tweaks needed before merging

ok I think all is good! lifting WIP

File: HACKING.md [hunk 1]
```diff
@@ -299,3 +310,38 @@ http://simple-httpd-fedora-coreos.$CLUSTER_URL
 ```
 
 (Or simply check the output of `oc get route simple-httpd`).
+
+
+
```
Comment (jlebon) [inline HACKING.md]: Minor: lots of extra blank spaces here

File: HACKING.md [hunk 2]
```diff
@@ -299,3 +310,38 @@ http://simple-httpd-fedora-coreos.$CLUSTER_URL
...
+## Running A Developer Pipeline Alongside an Existing Deployment
+
+If you already know of an existing deployed pipeline you can add some
+namespaced (i.e. DEVEL_PREFIX) resources into the existing project
+and run your own developer pipeline. This is useful if you simply want
+to test a different COSA image or FCOS config or pipeline code, but
+otherwise maintain the manifest the same.
+do this, you *must* use the `devel-up` script. For example:
```
Comment (jlebon) [inline HACKING.md]: Non sequitur

File: HACKING.md [hunk 3]
```diff
@@ -1,25 +1,18 @@
...
-may indicate that the section only applies to one of the two workflows.
+## Setting up a local or production cluster from scratch
 
-Another option if one simply wants to test a different COSA image or
-FCOS config or pipeline code (but otherwise maintain the manifest the
-same) is to create a developer pipeline alongside the production one. To
-do this, you *must* use the `devel-up` script. For example:
+The instructions in this section will cover setting up a local (developer)
+cluster as well as a production cluster pipeline in CentOS CI. If you
+have an existing deployment and just want to set up a parallel
```
Comment (jlebon) [inline HACKING.md]: Maybe instead of `have` --> `have access to` ?

File: HACKING.md (hunk 1 above)
Comment (dustymabe) [reply HACKING.md]: Yeah. didn't know if the extra spaces would help delineate the next heading as a new **major** section or not. Can you view this in rich format and let me know the best way to do that?

File: HACKING.md (hunk 2 above)
Comment (dustymabe) [reply HACKING.md]: yeah - copy/paste mistake I think - deleted in next upload.

File: HACKING.md (hunk 1 above)
Comment (dustymabe) [reply HACKING.md]: either way I deleted the extra space.

Comment (jlebon) [reply HACKING.md]: You can see it here: https://github.com/coreos/fedora-coreos-pipeline/blob/b2c9ec407ecb56b274c6b9377447c69c6e482080/HACKING.md.

AFAIK, markdown doesn't care about the number of spaces. Just using the appropriate number of `#` signs should be good. :+1:

**Comment by dustymabe** [comment]
new update ^^

**Comment by jlebon** [comment]
LGTM, thanks for documenting the new workflow!

**Pull request merged**